requirements/auto-lint.txt

```diff
@@ -0,0 +1,30 @@
+# ;; -*- mode: pip-requirements -*-
+
+autoflake
+autopep8
+docformatter
+flake8
+flake8-blind-except
+flake8-coding
+flake8-comprehensions
+flake8-debugger
+flake8-deprecated
+flake8-docstrings
+flake8-meiqia
+flake8-mutable
+flake8-pep3101
+flake8-print
+flake8-quotes
+flake8-string-format
+flake8-tidy-imports
+flake8-todo
```

Comment:
This `requirements.txt` file should be removed and use the `additional_dependencies` option for the `pre-commit` config files to install the "extra dependencies" for each of the plugins.